src/main/java/wolox/training/models/PasswordReset.java

```diff
@@ -0,0 +1,46 @@
+package wolox.training.models;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel
+public class PasswordReset {
+
+    @NotNull
+    @ApiModelProperty(required = true, notes = "Old password of the user")
+    private String oldPassword;
+
+    @NotNull
+    @ApiModelProperty(required = true, notes = "New password of the user")
+    private String newPassword;
+
+    public void setOldPassword(String oldPassword) {
+
+        checkNotNull(oldPassword, "Please check oldPassword field, its null");
+
+        this.oldPassword = oldPassword;
+
+    }
+
+    public void setNewPassword(String newPassword) {
+
+        checkNotNull(newPassword, "Please check newPassword field, its null");
+        checkArgument(!oldPassword.equals(newPassword), "Passwords must not be the same");
```

Comment:
I'm not sure if this method is being used. Anyway, nice verification, but the password may come empty and it won't any error in that case. Anyway this is fine so don't worry about that.